### PR TITLE
types for stream-to-blob

### PR DESCRIPTION
**Pull request by NetanelDavid**
Please fill in this template.

- [* ] Use a meaningful title for the pull request. Include the name of the package modified.
- [*] Test the change in your own code. (Compile and run.)
- [*] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [*] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [*] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [*] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

- [*] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [*] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [*] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [*] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [*] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [*] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/stream-to-blob/tslint.json
1. types/stream-to-blob/tsconfig.json
1. types/stream-to-blob/package.json
1. types/stream-to-blob/index.d.ts
1. types/stream-to-blob/stream-to-blob-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d41a5f02effd435f4e42c01847bbfccf1af7e8c5
</p>

File: types/stream-to-blob/stream-to-blob-tests.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/// <reference types = "node"/>
+
+import { Stream } from "stream";
+import streamToBlob from "stream-to-blob";
```
Comment (gabritto) [inline types/stream-to-blob/stream-to-blob-tests.ts:4]: I believe you should use the `import = ... ` syntax form here instead, to match the export syntax that was used:
`import streamToBlob = require("stream-to-blob");`

File: types/stream-to-blob/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "ES6"
+        ],
+        "esModuleInterop": true,
```
Comment (gabritto) [inline types/stream-to-blob/tsconfig.json:7]: Enabling `"esModuleInterop"` is non-standard on this repository, and should be avoided when possible.
Could you remove this option from this config?
I think if you also follow my recommendation on the import syntax used on the test, then there shouldn't be an issue.

**Review by gabritto — CHANGES_REQUESTED** [review]

File: types/stream-to-blob/tsconfig.json (hunk 2 above)
Comment (NetanelDavid) [reply types/stream-to-blob/tsconfig.json]: @gabritto thank for review, I fixed it as per your request.

File: types/stream-to-blob/stream-to-blob-tests.ts (hunk 1 above)
Comment (NetanelDavid) [reply types/stream-to-blob/stream-to-blob-tests.ts]: @gabritto thank for review, I fixed it as per your request.

File: types/stream-to-blob/stream-to-blob-tests.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+/// <reference types = "node"/>
+
+import { Stream } from "stream";
```
Comment (peterblazejewicz) [inline types/stream-to-blob/stream-to-blob-tests.ts:3]:
```suggestion
import { Stream } from "node:stream";
```

**Review by gabritto — APPROVED** [review]

**Comment by NetanelDavid** [comment]
Ready to merge

**Pull request merged**